pkg/core/hash.go

```diff
@@ -64,6 +64,38 @@ func calculateConfigHash(children []Object) (string, error) {
 	return fmt.Sprintf("%x", hashBytes), nil
 }
 
+// getConfigMapData extracts all the relevant data from the ConfigMap, whether that is
+// the whole ConfigMap or only the specified keys.
+func getConfigMapData(child configObject) map[string]string {
+	cm := corev1.ConfigMap(*child.object.(*corev1.ConfigMap))
```

Comment:
Don't need the cast after the type assertion
```suggestion
	cm := *child.object.(*corev1.ConfigMap)
```

Reply:
Fixed